R/tune_bayes.R

```diff
@@ -432,6 +432,12 @@ tune_bayes_workflow <-
             ~ dplyr::mutate(., .config = paste0("Iter", i))
           )
         }
+        if (".extracts" %in% names(tmp_res)) {
+          tmp_res[[".extracts"]] <- purrr::map(
+            tmp_res[[".extracts"]],
+            ~ dplyr::mutate(., .config = paste0("Iter", i))
+          )
+        }
```

Comment:
The substance of the PR is here and in the untouched lines above; we just need to make the same `mutate()` happen in `.extracts` that happened already for `.metrics` and `.predictions`.